app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ProductListsActivity.java

```diff
@@ -145,4 +164,99 @@ public void onRightClicked(int position) {
         adapter.notifyItemRangeChanged(position, adapter.getItemCount());
     }
 
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu_product_lists, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        int id = item.getItemId();
+        if (id == R.id.action_import_csv) {
+            selectCSVFile();
+        }
+        return super.onOptionsItemSelected(item);
+    }
+
+    private void selectCSVFile() {
+        Intent intent = new Intent(Intent.ACTION_GET_CONTENT);
+        intent.addCategory(Intent.CATEGORY_OPENABLE);
+        intent.setType("text/csv");
+        startActivityForResult(Intent.createChooser(intent, "Open CSV"), ACTIVITY_CHOOSE_FILE);
+    }
+
+    @SuppressLint("StaticFieldLeak")
+    class parseCSV extends AsyncTask<Void, Integer, Boolean> {
```

Comment:
It's good practice to use a Capital at the start of a class name. Please rename this to ParseCVS